plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -127,7 +127,20 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	}
 
 	trace.Step("Selecting host")
-	return g.selectHost(priorityList)
+	host, err := g.selectHost(priorityList)
+	if err != nil {
+		return "", err
+	}
+
+	if len(g.extenders) != 0 {
+		for _, extender := range g.extenders {
```

Comment:
I'm thinking to implement `Binder` interface; HTTPBinder & DefaultBinder (call apiserver directly). We can put implementation in `HTTPExtender.go`.